sample/particles/main.ts

```diff
@@ -330,6 +336,11 @@ const simulationUBOBuffer = device.createBuffer({
 const gui = new GUI();
 gui.add(simulationParams, 'simulate');
 gui.add(simulationParams, 'deltaTime');
+const hdrFolder = gui.addFolder('HDR Settings');
+hdrFolder
+  .add(simulationParams, 'toneMappingMode', ['standard', 'extended'])
+  .onChange(configureContext);
+hdrFolder.add(simulationParams, 'brightnessFactor', 0, 4, 0.1);
 
```

Comment:
```suggestion
hdrFolder.open();
```

wdyt? this will start with folder open